### PR TITLE
DOCS: Adding missing objects and updating privileges lists for some objects

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**
Changes are staging here (VPN required): https://jennyprivileges.sc2-04-pcf1-apps.oc.vmware.com/7-0/admin_guide/roles_privs.html

### CODE REVIEW

File: gpdb-doc/dita/admin_guide/roles_privs.xml [hunk 1]
```diff
@@ -310,17 +312,62 @@
...
+                  </p>
+                </entry>
+            </row>
             <row>
               <entry colname="col1">External Tables</entry>
               <entry colname="col2">
-                <codeph>SELECT</codeph>
+                <p>
+                  <codeph>SELECT</codeph>
+                </p>
```
Comment (jimmyyih) [inline gpdb-doc/dita/admin_guide/roles_privs.xml:370]: External tables are treated like regular tables for ACLs in the internal code.  You can probably remove this section and add "External Tables" to the "Tables, Views" section.

File: gpdb-doc/dita/admin_guide/roles_privs.xml [hunk 2]
```diff
@@ -347,19 +432,45 @@
...
+                <p>
+                  <codeph>UPDATE</codeph>
+                </p>
+                <p>
+                  <codeph>ALL</codeph>
+                </p>
               </entry>
             </row>
```
Comment (jimmyyih) [inline gpdb-doc/dita/admin_guide/roles_privs.xml:467]: Note sure if we want to add Large Objects to the doc.  We don't support large objects in GPDB AFAIK.

Comment (VMWareJennyBeth) [reply gpdb-doc/dita/admin_guide/roles_privs.xml:467]: Thanks for catching that, Jimmy! I'd forgotten that I'd dropped that in! Indeed, that type is just supported in Postgres, not in GPDB.

**Review by jimmyyih — APPROVED** [review]
Looks good IMO.  Just make sure to squash your commits before merging.

**Pull request merged**